test/core/realtime/hotelClerk/addSubscription.test.js

```diff
@@ -169,12 +169,36 @@ describe('Test: hotelClerk.addSubscription', () => {
       collection: 'bar',
       controller: 'realtime',
       action: 'join',
-      body: {roomId: 'no way I can exist'}
+      body: {roomId: 'i-exists'}
     }, context);
 
-    return should(() => hotelClerk.join(joinRequest)).throw(NotFoundError, {
-      id: 'core.realtime.room_not_found'
-    });
+    return should(hotelClerk.join(joinRequest))
+      .be.rejectedWith(NotFoundError, {
+        id: 'core.realtime.room_not_found'
+      });
+  });
+
+  it('#join should propagate notification only with "cluster" option', async () => {
```

Comment:
```suggestion
  it('#join should propagate notification only with "replicate" option', async () => {
```

Reply:
The option is named `cluster` for channel because it has more meaning to have "cluster" channels and not "replicate" channels